### DISCUSSION TITLE
[Kavita] Option to group libraries per country (or other upper level hierarchy)

### DISCUSSION THREAD

**Discussion by janaxhell** [Kareadita/Kavita#3063] [Ideas]
### Idea Description

Assuming one has comics sorted as (country)\publisher\series\issues 123.cbx
I have comics from many countries, mostly sorted by publisher, but this leads to ambiguities, because some publishers like Marvel, Disney or others are present in different countries publishing sometimes the same things in different languages, most of the times completely different things. Also, many publishers release translated publications from other countries, so that often series from the same original publisher are split between different publishers of another country. Having the chance to group publishers to USA, UK, France, Italy, Japan, etc would allow to immediately access to the correct version of a publication from the left side column. 
For example

France\Dargaud
Italy\Bonelli
Italy\Disney
Italy\Marvel
UK\Marvel
USA\DC
USA\Disney
USA\Marvel

As of now I have to name them like 

Disney Italia (which is actually Mondadori, Disney and a few others)
Disney USA (which is actually Dell\Gold Key and many others)
Marvel Italia (which is actually Panini, Star Comics, Play Press, Editoriale Corno and many others)
Marvel UK
Marvel USA 

and they are all mixed by the alphabetical order.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
You could probably setup smart filters. Library would be your Country, then smart filters will check against Path for the Publisher (FolderPath contains "Marvel") then you can name it Marvel (US).

**Comment by janaxhell** [comment]
This is interesting, but:
- is there any way to not have a clicked filter open in a new tab?
- is there any way to sort alphabetically the smart filters? I only see the option to delete them.

**Reply by majora2007** [reply]
You can sort smart filters in any order you want when you bind them to your side nav or dashboard. You can turn them on or off there's complete flexibility. 

Smart filters of course can also have a custom sort order and field as well. So if you don't want your series to be alphabetical for some strange reason, then you could.

**Comment by janaxhell** [comment]
Ahh! I didn't see the Smart Filters in the Nav Bar Customize tab because I was rolling the mouse wheel and only the Libraries entries were showing, while Smart Filters were "hidden" in the lower part of the tab, out of my mouse wheel reach. It came up by chance while I was moving the mouse around outside the Libraries frame. Thanks, this is very close to what I meant.
